### ISSUE TITLE
Refresh db

### ISSUE THREAD

**Issue by marlycormar** [pharmaR/risk_assessment#135]
Add a mechanism to refresh the db:
- Update metrics for each package: more recent number of downloads, more recent bug closure percentage, etc.
- Do not update the package version.
- Leave a comment that the package has been updated. 
- Add somewhere on the tabs the last day the db was refreshed.
- List all the changes in the modal that will take place if admin updates the risk per package. Put a table in the modal of which packages will need to be reviewed after the risk is updated.

**Comment by marlycormar** [comment]
I don't think this is a good idea. We should incorporate package versions. Doing so, we will get rid of issues with out-of-date packages.

**Comment by AARON-CLARK** [comment]
I still see value in this proposed feature. It basically a button that mimics exactly what the "re-calculate risk" button does, but doesn't require any weights be changed. There are so many metrics that can become stale if they were loaded into the app a long time ago. Like what if I'm evaluating a package based on how it looked 6 months ago? It's score could be totally different today.

I propose a compromise: Why don't we just allow admin users to click the "Re-calculate Risk" button even when they haven't changed the weights? Thoughts?

**Comment by AARON-CLARK** [comment]
whoops, I should have stated: a condition of making this compromise work would be allowing the admin to select which packages get updated, incorporating the feature request proposed in #145.

**Comment by AARON-CLARK** [comment]
Re-opening this issue for consideration at a later date, especially with the idea #253 proposes.

**Comment by aclark02-arcus** [comment]
Potential duplicate of #493
